en/micro-integrator/docs/references/connectors/ldap-connector/ldap-connector-example.md

```diff
@@ -0,0 +1,214 @@
+    <property expression="json-eval($.providerUrl)" name="providerUrl" scope="default" type="STRING"/>
+    <property expression="json-eval($.securityPrincipal)" name="securityPrincipal" scope="default" type="STRING"/>
+    <property expression="json-eval($.securityCredentials)" name="securityCredentials" scope="default" type="STRING"/>
+    <ldap.init>
+        <providerUrl>{$ctx:providerUrl}</providerUrl>
+        <securityPrincipal>{$ctx:securityPrincipal}</securityPrincipal>
+        <securityCredentials>{$ctx:securityCredentials}</securityCredentials>
+        <secureConnection>{$ctx:secureConnection}</secureConnection>
+        <disableSSLCertificateChecking>{$ctx:disableSSLCertificateChecking}</disableSSLCertificateChecking>
+    </ldap.init>
+</sequence>
+```
+
+* add_student_sequence - `<ldap.addEntry>` element creates a new LDAP entry in the LDAP server
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<sequence name="add_student_sequence" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+    <property expression="json-eval($.content.objectClass)" name="objectClass" scope="default" type="STRING"/>
+    <property expression="json-eval($.content.attributes)" name="attributes" scope="default" type="STRING"/>
+    <property expression="json-eval($.content.dn)" name="dn" scope="default" type="STRING"/>
+    <ldap.addEntry>
+        <objectClass>{$ctx:objectClass}</objectClass>
+        <attributes>{$ctx:attributes}</attributes>
+        <dn>{$ctx:dn}</dn>
+    </ldap.addEntry>
+    <respond/>
+</sequence>
+```
+* search_student_sequence - `<ldap.searchEntry>` element search for one or more LDAP entities based on the specified 
+search keys.
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<sequence name="search_student_sequence" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+    <property expression="json-eval($.content.objectClass)" name="objectClass" scope="default" type="STRING"/>
+    <property expression="json-eval($.content.filters)" name="filters" scope="default" type="STRING"/>
+    <property expression="json-eval($.content.attributes)" name="attributes" scope="default" type="STRING"/>
+    <property expression="json-eval($.content.dn)" name="dn" scope="default" type="STRING"/>
+    <ldap.searchEntry>
+        <objectClass>{$ctx:objectClass}</objectClass>
+        <limit>1000</limit>
+        <filters>{$ctx:filters}</filters>
+        <dn>{$ctx:dn}</dn>
+        <attributes>{$ctx:attributes}</attributes>
+    </ldap.searchEntry>
+    <respond/>
+</sequence>
+```
+
+{!references/connectors/exporting-artifacts.md!}
+
+## Deployment
+
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+
+## Testing
+
+### Create an entry in ldap server
+
+1. Create a file named student_data.json with following sample payload
```

Comment:
```suggestion
1. Create a file named student_data.json with following sample payload.
```

Does it need to be created in a specific location? If so, shall we mention that as well?

Reply:
Fixed